mpp/include/mpp_util.inc

```diff
@@ -1447,6 +1447,7 @@ end function rarray_to_char
                             '.  (IOSTAT = '//trim(text)//')')
           else
              num_lines = 1
+             max_length = 1
```

Comment:
If we always have num_lines and max_length > 0 for any successfully opened file, how does a user know when the file is truly empty?  Could the hang have been avoided with better error checking of the results of the function?

Reply:
The output of `get_ascii_file_num_lines_and_length` is going to be 1 more than the actual number of lines and lengths. 

So an empty file will output 1 for num_lines and max_length to prevent an allocation of 0. 

I suppose we can add error checking here in `read_input_nml`
https://github.com/NOAA-GFDL/FMS/blob/caf4466241698a41e3272becbc2ed419a8d3c248/fms2_io/fms_io_utils.F90#L496-L497

and here in `ascii_reads`

https://github.com/NOAA-GFDL/FMS/blob/caf4466241698a41e3272becbc2ed419a8d3c248/mpp/include/mpp_util.inc#L1332-L1333

Reply:
@uramirez8707 - what do you think this is best solution - error checking?  Or max_length=num_lines = 1?

Reply:
@bensonr 

I think we should leave them as `max_length=num_lines = 1`

I think it is cleaner than having to check for 0s every time `get_ascii_file_num_lines_and_length` is called.